man/portablectl.xml

```diff
@@ -155,6 +155,20 @@
         to be used in case the unit names do not match the image name as described in the <command>attach</command>.</para>
       </varlistentry>
 
+      <varlistentry>
+        <term><command>reattach</command> <replaceable>IMAGE</replaceable> [<replaceable>PREFIX…</replaceable>]</term>
+
+        <listitem><para>Detaches an existing portable service image from the host, and immediately attaches the new
+        specified version. Running units are not stopped during the process. Partial matching, to allow for different
```

Comment:
"new specified version"? there's no "version" specified here.

Just maybe say "Detaches an existing portable service image from the host, and immediately attaches it again. This is useful in case the image was replaced" or so

Reply:
Leftover from when the api was called 'upgrade' - changed as suggested. The idea btw as you saw later is that matching is relaxed - so doing 'reattach image_0.2.raw' will match an existing attached image_0.1.raw